opensearchpy/_async/client/indices.py

```diff
@@ -104,9 +104,7 @@ async def flush(self, index=None, params=None, headers=None):
             "POST", _make_path(index, "_flush"), params=params, headers=headers
         )
 
-    @query_params(
-        "include_type_name", "master_timeout", "timeout", "wait_for_active_shards"
-    )
+    @query_params("master_timeout", "timeout", "wait_for_active_shards")
```

Comment:
While we are working on this, is there a plan to work on inclusive naming for clients?
I've opened up an issue as I didn't see one: https://github.com/opensearch-project/opensearch-clients/issues/16

Reply:
The inclusive naming changes will be done once the breaking changes are in for the clients. I will add a reference to this issue in the original meta issue.